package.json

```diff
@@ -59,11 +59,16 @@
     "autoprefixer": "9",
     "classnames": "2.3.1",
     "craco-alias": "3.0.1",
+    "cypress": "9.0.0",
+    "cypress-dotenv": "^2.0.0",
+    "dotenv": "^10.0.0",
     "dotenv-cra": "2.0.2",
     "eslint": "7.24.0",
     "eslint-config-react-config-r13v": "1.4.0",
+    "eslint-plugin-jest": "^25.2.4",
     "eslint-plugin-react": "7.23.2",
     "husky": "7.0.2",
+    "jest": "^26.6.0",
```

Comment:
Did you add jest intentionally?

Reply:
I added it when was fighting with error from eslint. Guess it is not necessary